### ISSUE TITLE
unclear explanation in the Text classification with TensorFlow Hub: Movie reviews example

### ISSUE THREAD

**Issue by huikang** [tensorflow/tensorflow#38091]
## URL(s) with the issue:

https://www.tensorflow.org/tutorials/keras/text_classification_with_hub#build_the_model

Please provide a link to the documentation entry, for example:

https://www.tensorflow.org/tutorials/keras/text_classification_with_hub#build_the_model

## Description of issue (what needs changing):

The last layer in the model is `model.add(tf.keras.layers.Dense(1))`. However, in the following description, it says `The last layer is densely connected with a single output node. Using the sigmoid activation function, ...`

I check the api doc and find that the default activation is none for dense layer. So shall the example code be `model.add(tf.keras.layers.Dense(1, activation='sigmoid'))`?

**Comment by maxkaustav** [comment]
fixing this issue
